### PR TITLE
980 log received events not emitted

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
We were event logging the emitted events, some of this was partially fixed randomly in other tickets.

# What has changed
EventProcessor only logs received events, no emitted ones
The test for date serialisation in the database was removed from SampleReceiverIT as no longer appropriate, similar test placed in receiptingIT instead.

# How to test?
maven clean install

# Links
https://trello.com/c/9drX7XPj/980-logging-the-wrong-things-in-the-case-events-table
# Screenshots (if appropriate):

### CODE REVIEW

File: src/test/java/uk/gov/ons/census/casesvc/logging/EventLoggerTest.java [hunk 1]
```diff
@@ -34,6 +28,7 @@
 public class EventLoggerTest {
 
   private static final UUID TEST_CASE_ID = UUID.randomUUID();
+  public static final String TEST_JSON = "{test: json}";
```
Comment (NickGrantONS) [inline src/test/java/uk/gov/ons/census/casesvc/logging/EventLoggerTest.java]: This isn't JSON

**Comment by AdeHarrison** [comment]
as you're in there, maybe get rid of some of the duplicate code by making https://github.com/ONSdigital/census-rm-case-processor/blob/0ca97be5d9e5db89708754bbe1b71462c348b7bc/src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java#L47-L51 into a single helper method?

File: src/main/java/uk/gov/ons/census/casesvc/model/entity/EventType.java [hunk 2]
```diff
@@ -8,5 +8,6 @@
   RESPONSE_RECEIVED,
   REFUSAL_RECEIVED,
   FULFILMENT_REQUESTED,
-  QUESTIONNAIRE_LINKED
+  QUESTIONNAIRE_LINKED,
+  SAMPLE_LOADED;
```
Comment (AdeHarrison) [inline src/main/java/uk/gov/ons/census/casesvc/model/entity/EventType.java]: as picky as you can get but don't really need ;

File: src/test/java/uk/gov/ons/census/casesvc/logging/EventLoggerTest.java (hunk 1 above)
Comment (AdeHarrison) [inline src/test/java/uk/gov/ons/census/casesvc/logging/EventLoggerTest.java]: could be private

File: src/test/java/uk/gov/ons/census/casesvc/service/EventProcessorTest.java [hunk 3]
```diff
@@ -2,11 +2,9 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.eq;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (AdeHarrison) [inline src/test/java/uk/gov/ons/census/casesvc/service/EventProcessorTest.java]: optimise imports

File: src/test/java/uk/gov/ons/census/casesvc/service/EventProcessorTest.java [hunk 4]
```diff
@@ -2,11 +2,9 @@
...
+import static uk.gov.ons.census.casesvc.service.EventProcessor.*;
```
Comment (AdeHarrison) [inline src/test/java/uk/gov/ons/census/casesvc/service/EventProcessorTest.java]: ptimise imports

File: src/test/java/uk/gov/ons/census/casesvc/service/ReceiptProcessorTest.java [hunk 5]
```diff
@@ -2,12 +2,11 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (AdeHarrison) [inline src/test/java/uk/gov/ons/census/casesvc/service/ReceiptProcessorTest.java]: optimise imports

**Pull request merged**